### DISCUSSION TITLE
App Store Connect API 1.7 - Features Request

### DISCUSSION THREAD

**Discussion by kashitamang** [fastlane/fastlane#21948] [Ideas]
Wondering is there any Fastlane support for the following App Store Optimization features from the 2.0 release of the App Store Connect API? These are super powerful tools but can take hours to build manually, especially across multiple localizations. The AppEvents endpoints GET/DELETE/POST are working so far! 

💡[App Events](https://developer.apple.com/documentation/appstoreconnectapi/app_store/app_events_and_metadata/app_events)
💡[Custom Product Page](https://developer.apple.com/documentation/appstoreconnectapi/app_store/custom_product_pages_and_localizations)
💡[App Store Version Experiments](https://developer.apple.com/documentation/appstoreconnectapi/app_store/app_store_version_experiments)

**Comment by JDVSCM** [comment]
I agree!

**Comment by jj4rnn** [comment]
CPP, Events and Experiments -  It will be much faster to set up CPP, Events, and Experiments if we include them in the Fastlane route.
